proto/frequenz/api/common/common.proto

```diff
@@ -0,0 +1,378 @@
+// Contains enums and messages reused throught the Microgrid API definitions.
+//
+// Copyright:
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// License:
+// MIT
+
+syntax = "proto3";
+
+package frequenz.api.common.common;
+
+// A set of lower and upper bounds for any metric.
+// The units of the bounds are always the same as the related metric.
+message Bounds {
```

Comment:
We should remove everything except `DC` and `AC`, and rename this file to `electrical.proto`. Everything else is specific to the Microgrid API.